lib/vinyldns/api.rb

```diff
@@ -53,7 +53,7 @@ def self.make_request(signed_object, uri, body = '')
       url = URI(signed_object.api_url)
       https = Net::HTTP.new(url.host, url.port)
       https.use_ssl = true ? url.scheme == "https" : https.use_ssl = false
-      https.verify_mode = OpenSSL::SSL::VERIFY_NONE # SSL not signed? Don't care!
+      https.verify_mode = ENV['VINYLDNS_VERIFY_SSL'] == false || !ENV['VINYLDNS_VERIFY_SSL'].nil? ? OpenSSL::SSL::VERIFY_NONE : OpenSSL::SSL::VERIFY_PEER
```

Comment:
can this just be `ENV['VINYLDNS_VERIFY_SSL'] == "false" ? OpenSSL::SSL::VERIFY_NONE : OpenSSL::SSL::VERIFY_PEER`? The envoirnment variable will be a string, right now the `ENV['VINYLDNS_VERIFY_SSL'] == false` is always returning false so it's the `!ENV['VINYLDNS_VERIFY_SSL'].nil?` that's catching anything a user sets as the variable.

Reply:
oops didn't refresh after your most recent commit